src/win/util.c

```diff
@@ -1466,6 +1466,65 @@ int uv_os_setpriority(uv_pid_t pid, int priority) {
   return r;
 }
 
+/**
+ * If the function succeeds, the return value is 0.
+ * If the function fails, the return value is non-zero.
+*/
+int uv_thread_getpriority(uv_thread_t tid, int* priority) {
+  int r;
+
+  if (priority == NULL)
+    return UV_EINVAL;
+
+  r = GetThreadPriority(tid);
+  if (r == THREAD_PRIORITY_ERROR_RETURN) {
+    r = uv_translate_sys_error(GetLastError());
+  } else {
+    *priority = r;
+    r = 0;
+  }
+  return r;
```

Comment:
Somewhat unidiomatic for libuv code. Prefer this style:
```suggestion
  if (r == THREAD_PRIORITY_ERROR_RETURN)
    return uv_translate_sys_error(GetLastError());

  *priority = r;
  return 0;
```